openaerostruct/docs/_exts/embed_n2.py

```diff
@@ -0,0 +1,106 @@
+from docutils import nodes
```

Comment:
Where is this coming from? Are we just copying code from OM?

Reply:
When we removed the OM dependency, this was the only extension we couldn't replace with a default sphinx directive.  Embedding raw html into the docs caused several formatting errors that I couldn't fix given the default sphinx options.  This is the same approach taken by `openconcept`, but I think we should strongly consider moving to Jupyter based docs to eliminate all these issues.

Reply:
I would prefer a comment in the file explaining where this code is from, and perhaps link to the file on GitHub (from the version where this was taken).

Reply:
I agree that is a good idea.  I will add it now